### DISCUSSION TITLE
Extractor confirmation please

### DISCUSSION THREAD

**Discussion by orddie** [dcecchino/glog#16] [General]
Making sure I get this right. 
vmware_esxi_extractors = ESXi input
vmware7_extraction = vCenter input

is that correct?

**Comment by dcecchino** [comment]
For vcenter 6.5 you can apply both extractor to the same input but for vcenter 7, you have the correct method.On Feb 18, 2023, at 12:24 PM, orddie ***@***.***> wrote:﻿
Making sure I get this right.
vmware_esxi_extractors = ESXi input
vmware7_extraction = vCenter input
is that correct?
